### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by Nite16** [tjmarkham/win10widgets#46]
Hate to be "that guy", but curious if this project is abandoned as there's been no activity in 3 months?

**Comment by leonardosegurat** [comment]
He might be overwhelmed by work/school at the moment. Hopefully he'll come back. Either way, we could post our own workarounds/new features while we wait for updates.
I messed around a bit with the code to make a kind-of working multi-timezone support. *sigh* If only I knew more about coding...

**Comment by xankazo** [comment]
It has been several months and no updates. Maybe it is.

**Comment by tjmarkham** [comment]
Thanks for your interest guys! Win10 Widgets isn't abandoned, but it's definitely on "pause" right now due to time constraints.

When I released Win10 Widgets I was working on it full-time since I didn't have a job. Now that I've graduated, moved in with my girlfriend, and am working a full-time job, I have a lot less time I can afford to allocate to Win10 Widgets. I still use Win10 Widgets every day and I'll 100% get back to development work eventually, but right now the time just isn't there; work, family, etc. has to come first.

In the meantime, people can free to contribute! Thanks again for checking out Win10 Widgets, and stay tuned for future updates; they'll come eventually :)
